src/frontend/pages/game.tsx

```diff
@@ -100,6 +110,14 @@ export default function Home(): ReactElement {
     return () => clearInterval(interval);
   }, [timeLeft]);
 
+  // make this not show up twice
+  if (timeLeft <= 3 && !takeComplete) {
+    toast.notify('Time is Running Out!', {
+      duration: 3000,
```

Comment:
Question on why this one is a different amount of time than the others

Reply:
The goal here was to have it last 3 seconds (when there's 3 seconds left on the timer). Right now that part is working but instead of one lasting 3 seconds, it's showing 3 different toasts

Reply:
thats because its calling anytime it's less than three. You want it to just be when it equals three

Reply:
yep made this change accordingly, thank you